lib/fixme.ex

```diff
@@ -2,20 +2,26 @@ defmodule FIXME do
   defmacro fixme(
     {:-, _, [{:-, _, [year, month]}, day]},
     message,
-    opts \\ :default_opts
+    opts \\ []
   ) do
     {opts, _} = Code.eval_quoted(opts)
 
     # Injectable "today" for tests.
-    current_date = case opts do
-      :default_opts -> :calendar.local_time |> elem(0)
-      [today: date] -> date
-    end
+    current_date = opts[:today] || (:calendar.local_time |> elem(0))
+
+    # Opt into compile time warnings
+    warn = opts[:warn]
```

Comment:
Not a real use case, but anybody using the project will suddenly start getting warnings on updating the dependencies. Hence went with off by default.